stable/traefik/templates/configmap.yaml

```diff
@@ -34,6 +34,14 @@ data:
           KeyFile = "/ssl/tls.key"
       {{- end }}
     [kubernetes]
+    {{- if .Values.kubernetes}}
+      {{- if .Values.kubernetes.namespaces }}
+    namespaces = {{ .Values.kubernetes.namespaces }}
```

Comment:
Should range over the YAML list here to create a TOML list as the value for `namespaces`.